src/encoded/commands/deploy.py

```diff
@@ -4,22 +4,19 @@
 - SpotClient was removed in EPIC-ENCD-4716/ENCD-4688-remove-unused-code-from-deploy.
 """
 import argparse
-import datetime
 import getpass
 import io
 import re
 import subprocess
 import sys
-import time
 
-from base64 import b64encode
 from difflib import Differ
 from os.path import expanduser
 
 import boto3
 
 
-def nameify(in_str):
+def _nameify(in_str):
```

Comment:
I can't tell if anyone of the changes in deploy.py are functional at all, or just style/lint changes.

Reply:
Commit history.